auto-pos-creator/tests/pos_creator_setup/mod.rs

```diff
@@ -67,6 +67,20 @@ where
     MetastakingBuilder: 'static + Copy + Fn() -> farm_staking_proxy::ContractObj<DebugApi>,
     PosCreatorBuilder: 'static + Copy + Fn() -> auto_pos_creator::ContractObj<DebugApi>,
 {
+    // Pairs setup:
+    // 3 pools (A, B), (A, C), (B, C),
+    // A:B 1:2
+    // A:C 1:6
+    // B:C 1:3
+
+    // Pools: (B = billion)
+    // (A, B) => (1B, 2B)
+    // (A, C) => (1B, 6B)
+    // (B, C) => (1B, 3B)
+    //
+    // A_total = 2B
+    // B_total = 3B
+    // C_total = 9B
```

Comment:
I think here it would be C_total = 6B?

Reply:
Nope, 6B from the (A, C) pool + 3B from (B, C) pool

Reply:
Something still seems off. I don't really understand to what you are referring to with those numbers.

Reply:
Total number of tokens, so I know the total overall balance I'll need to set on line 133 for the owner's balance.